pkgs/by-name/an/any-nix-shell/package.nix

```diff
@@ -0,0 +1,54 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchpatch2,
+  makeWrapper,
+  bash,
+  gnused,
+  which,
+}:
+
+stdenv.mkDerivation {
+  pname = "any-nix-shell";
+  version = "1.2.1-unstable-2023-11-08";
+
+  src = fetchFromGitHub {
+    owner = "haslersn";
+    repo = "any-nix-shell";
+    rev = "2537e5c6901ef934f8f44d61bcfe938b0fc9fa71";
+    hash = "sha256-j1DE0WTBGLmBLoPmqST9YVj9Jc4Mp8WXQILmPBzRlbM=";
+  };
+
+  patches = [
+    # https://github.com/haslersn/any-nix-shell/pull/34
+    (fetchpatch2 {
+      name = "add_support_for_the_nix_develop_command.patch";
+      url = "https://github.com/haslersn/any-nix-shell/commit/f048649700a047150d3bbc399869c4e003c96125.patch";
+      hash = "sha256-Bl4akNy3Aj7LkHTnKOsIdYbIc8LvFH7rjg89QPoLaYk=";
+    })
+  ];
+
+  strictDeps = true;
+  nativeBuildInputs = [ makeWrapper ];
+  buildInputs = [ bash ];
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -r bin $out
+    wrapProgram $out/bin/any-nix-shell --prefix PATH ":" ${
+      lib.makeBinPath [
+        (placeholder "out")
+        gnused
+        which
+      ]
+    }
+  '';
+
+  meta = with lib; {
+    description = "fish, xonsh and zsh support for nix-shell";
+    license = licenses.mit;
+    homepage = "https://github.com/haslersn/any-nix-shell";
+    maintainers = with maintainers; [ haslersn ];
```

Comment:
Is theey active??

Reply:
Not sure, but I don't think I should remove it in this PR.

Maybe a dedicated PR to remove them.

Reply:
Yes, I'm active